Makefile

```diff
@@ -19,11 +19,13 @@ endif
 export CC T_DIR ROOT_DIR CCFLAGS LDFLAGS
 all:$(SUBDIRS) DEBUG
 $(SUBDIRS):ECHO
-	mkdir -p $(T_DIR)/src
+	mkdir -p $(T_DIR)/IOH
```

Comment:
`IOH` contains the header files, correct? If yes, please use the standard name `include`